diff --git a/external/openmp/Makefile b/external/openmp/Makefile
--- a/external/openmp/Makefile
+++ b/external/openmp/Makefile
@@ -33,9 +33,9 @@
 
 OMP_DIR = openmp_code
 LIBOMP = $(OMP_DIR)/final/build/runtime/src/libomp.a
 
-OMP_CONFIG = -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER_WORKS=TRUE -DCMAKE_CXX_COMPILER_WORKS=TRUE
+OMP_CONFIG = -DCMAKE_POLICY_VERSION_MINIMUM=3.5 -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER_WORKS=TRUE -DCMAKE_CXX_COMPILER_WORKS=TRUE
 ifdef DEBUG
 	OMP_CONFIG = -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER_WORKS=TRUE -DCMAKE_CXX_COMPILER_WORKS=TRUE
 endif
 ENC_CXXFLAGS = $(patsubst -fpie, -fpic, $(ENCLAVE_CXXFLAGS))
